src/data/polynomial/algebra_map.lean

```diff
@@ -314,3 +324,27 @@ begin
 end
 
 end polynomial
+
+section alg_hom
+
+open polynomial
+
+variables {R A A' : Type*}
+
+section comm_semiring
+
+variables [comm_semiring R] [comm_semiring A] [algebra R A] [comm_semiring A'] [algebra R A']
+
+@[simp]
+lemma alg_hom.map_aeval (e : A →ₐ[R] A') (x : A) (f : polynomial R) :
+  e (aeval x f) = aeval (e x) f :=
+by simp [aeval_eq_sum_range, e.map_sum]
```

Comment:
Isn't this just `polynomial.aeval_alg_hom_apply` stated in reverse?

Reply:
You're right! I really did my best to find this result because it should be there, turns out it is :/

Reply:
The `alg_equiv` version is probably still worth having

Reply:
You're right! I should really not check my e-mails when I am too tired to think...